src/Combinatorics/Graphs/functions.jl

```diff
@@ -1339,3 +1339,20 @@ function laplacian_matrix(g::Graph)
   A = matrix(ZZ, adjacency_matrix(g))
   return D-A
 end
+
+@doc raw"""
+    is_bipartite(g::Graph)
+
+Returns true if the graph `g`  is bipartite.
+
+# Examples
+```
+julia> g = graph_from_edges([1,2],[2,3],[3,4]]);
+
+julia> is_bipartite(G)
+true
+```
+"""
+function is_bipartite(g::Graph{T}) where T
+  return Polymake.graph.Graph{T}(ADJACENCY=g.pm_graph).BIPARTITE::Bool
```

Comment:
```suggestion
function is_bipartite(g::Graph{Undirected})
  return Polymake.graph.Graph{Undirected}(ADJACENCY=pm_object(g)).BIPARTITE::Bool
```
This is only defined for undirected graphs.

```
graph/objects/Combinatorics/Graph/properties/BIPARTITE:
property BIPARTITE : Bool
 Only defined for Graph<Undirected>.
 True if the graph is a bipartite.
```